code/game/machinery/computer/_computer.dm

```diff
@@ -87,6 +87,16 @@
 		playsound(loc, 'sound/effects/glassbr3.ogg', 100, TRUE)
 		set_light(0)
 
+/obj/machinery/computer/proc/imprint_gps(gps_tag) // Currently used by the upload computers and communications console
+	var/tracker = gps_tag
+	if(!tracker) // Don't give a null GPS signal if there is none
+		return
+	for(var/obj/item/circuitboard/computer/board in src.contents)
+		if(!contents || board.GetComponent(/datum/component/gps))
+			return
+		board.AddComponent(/datum/component/gps, "[tracker]")
+		balloon_alert_to_viewers("Board Authenticated and Imprinted", vision_distance = 1)
```

Comment:
```suggestion
		balloon_alert_to_viewers("board tracker enabled", vision_distance = 1)
```
or `board authenticated` or `board imprinted` or whatever else.
Convention with balloon alerts is that they should be entirely lower case, and also as short as reasonably possible.